modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheOperationContext.java

```diff
@@ -32,9 +32,12 @@
  */
 public class CacheOperationContext implements Serializable {
     /** */
-    //TODO IGNITE-8801 remove this and set default as `false`.
-    public static final boolean DFLT_ALLOW_ATOMIC_OPS_IN_TX =
-        IgniteSystemProperties.getBoolean(IGNITE_ALLOW_ATOMIC_OPS_IN_TX, true);
+    public static final boolean DFLT_ALLOW_ATOMIC_OPS_IN_TX = false;
+
+    /** */
+    public static final boolean defaultAllowAtomicOpsInTx() {
```

Comment:
Let's rename this method, as it doesn't return default value now

Reply:
thank you, I will rename it to allowAtomicOpsInTx()